### PR TITLE
jsonapi is deprecated, just use jsonapi-renderer

### PR DESCRIPTION
**Pull request by jaredbeck**
From the author of jsonapi:

> .. The jsonapi gem was previously just a bundle of jsonapi-serializer and jsonapi-renderer, and AMS is using only a helper class of jsonapi-renderer (namely JSONAPI::IncludeDirective). The AMS dependency was previously not properly pinned to a specific version, which I saw as a risk for many users, so I avoided updating this gem. Moreover, the name jsonapi being somewhat too generic for what this gem evolved into (namely jsonapi-rb, which bundles jsonapi-renderer and jsonapi-parser, along with serializers and deserializers, with tight integrations with various frameworks), I decided to stay away from it for fairness.
> TL;DR: Yes, people should use jsonapi-parser and jsonapi-renderer directly (or give a try to jsonapi-rb, depending on their needs).
> https://github.com/beauby/jsonapi/issues/54#issuecomment-280537900

We should also update jsonapi-renderer to the latest, currently 0.1.2, but I thought that should be a separate PR.

### CODE REVIEW

File: active_model_serializers.gemspec [hunk 1]
```diff
@@ -42,7 +42,8 @@ Gem::Specification.new do |spec|
   # 'minitest'
   # 'thread_safe'
 
-  spec.add_runtime_dependency 'jsonapi', '0.1.1.beta6'
+  # TODO: Latest jsonapi-renderer is 0.1.2
+  spec.add_runtime_dependency 'jsonapi-renderer', '0.1.1.beta1'
```
Comment (bf4) [inline active_model_serializers.gemspec]: oh cool. why lock it to this version?

Comment (jaredbeck) [inline active_model_serializers.gemspec]: i didn't want to change the version in this PR. i'd be happy to do another PR that changes the version constraint.

Comment (bf4) [inline active_model_serializers.gemspec]: ah, so that's the version in the gem. Sure, let's do this. Can you add a changelog under misc?

**Comment by jaredbeck** [comment]
> Can you add a changelog under misc?

Added.

**Comment by jaredbeck** [comment]
@bf4 Hi Benjamin, can you please release a new version with this change? Thanks!

**Comment by bf4** [comment]
Sure. I assume you're running on master? @jaredbeck

**Comment by jaredbeck** [comment]
> Sure. I assume you're running on master? @jaredbeck

No, but I can try master with my app's test suite, just a sec.

**Comment by bf4** [comment]
Me too, just passed.

**Comment by bf4** [comment]
Released 

B mobile phone

**Comment by jaredbeck** [comment]
My app's test suite (admittedly limited) passes on 0.10.5, FWIW. Thanks for the release, Benjamin.
